File: app/src/main/java/ds/vkplus/eventbus/events/ClickEvent.java
package ds.vkplus.eventbus.events;

import ds.vkplus.network.model.News;

public class ClickEvent {
	public long id;
	public News item;
	public int viewId;
	public String url;


	public ClickEvent(final long id, final int viewId) {
		this.id = id;
		this.viewId = viewId;
	}

	public ClickEvent(final News item, final int viewId) {
		this.item = item;
		this.viewId = viewId;
	}

	public ClickEvent(final String url, final int viewId) {
		this.url = url;
		this.viewId = viewId;
	}
}

File: app/src/main/java/ds/vkplus/eventbus/events/ClickEvent.kt
package ds.vkplus.eventbus.events

import ds.vkplus.model.News

class ClickEvent {
	var id: Long = 0
	lateinit var item: News
	var viewId: Int = 0
	lateinit var url: String
	
	
	constructor(id: Long, viewId: Int) {
		this.id = id
		this.viewId = viewId
	}
	
	constructor(item: News, viewId: Int) {
		this.item = item
		this.viewId = viewId
	}
	
	constructor(url: String, viewId: Int) {
		this.url = url
		this.viewId = viewId
	}
}
